nio/client/async_client.py

```diff
@@ -1121,6 +1145,100 @@ def add_response_callback(
         raise SendRetryError("Max retries exceeded while trying to send "
                              "the message")
 
+    @logged_in
+    async def room_put_state(
+            self,
+            room_id:    str,
+            event_type: str,
+            content:    Dict[Any, Any],
+            state_key:  str            = ""
+    ) -> Union[RoomPutStateResponse, RoomPutStateError]:
+        """Send a state event to a room.
+
+        Returns either a `RoomPutStateResponse` if the request was successful
+        or a `RoomPutStateError` if there was an error with the request.
+
+        Args:
+            room_id (str): The room id of the room to send the event to.
+            event_type (str): The type of the state to send.
+            content (Dict[Any, Any]): The content of the event to be sent.
+            state_key (str): The key of the state event to send.
+        """
+
+        method, path, data = Api.room_put_state(
+            self.access_token,
+            room_id,
+            event_type,
+            content,
+            state_key = state_key
+        )
+
+        return await self._send(
+            RoomPutStateResponse,
+            method,
+            path,
+            data,
+            response_data = (room_id,),
+        )
+
+    @logged_in
+    async def room_get_state(
+            self,
+            room_id:    str,
```

Comment:
There's a bunch of unnecessary white-space I assume that's from a copy paste. You don't have to fix this since we're gonna switch to black relatively soon™ and I'll tell people to just format code with black.